stacks/static/service-config/mime.types

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
Given that this is 100% repeated from `stacks/lemp/service-config/mime.types` I wonder if we can use a symlink in the `stacks` directory, and then make sure `vagrant-spk` copies the file fully into the app.